hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/conf/TestConfigurationFieldsBase.java

```diff
@@ -205,6 +205,10 @@
       if (configDebug) {
         System.out.println("  Value: " + value);
       }
+      
+      if (value == null) {
```

Comment:
Is that because of the `null` default alternative scheme?

Reply:
Yes, it is not needed anymore after switching from null to empty string. I removed it.